security/pfSense-pkg-acme/files/usr/local/pkg/acme/acme.sh

```diff
@@ -0,0 +1,4587 @@
+#!/usr/bin/env sh
```

Comment:
@PiBa-NL INSTALL_DATA macro already set -m 0644, add it explicit is redundant. And for scripts you should use INSTALL_SCRIPT macro and remove -m 0755